pymongo/asynchronous/pool.py

```diff
@@ -1473,12 +1483,13 @@ def __init__(
         self.address = address
         self.opts = options
         self.handshake = handshake
-        # Don't publish events in Monitor pools.
+        # Don't publish events or logs in Monitor pools.
         self.enabled_for_cmap = (
             self.handshake
             and self.opts._event_listeners is not None
             and self.opts._event_listeners.enabled_for_cmap
         )
+        self.enabled_for_logging = self.handshake and not self.opts.is_monitor
```

Comment:
is_monitor is a duplicate of handshake. handshake is false for monitor pools and True for app pools. I suggest removing it.